var/spack/repos/builtin/packages/sed/package.py

```diff
@@ -25,6 +25,8 @@ class Sed(AutotoolsPackage, GNUMirrorPackage):
 
     executables = ["^sed$"]
 
+    tags = ["build-tools"]
```

Comment:
Similarly, we can extract to a separate PR adding the `build-tools` tag to existing packages